### ISSUE TITLE
Footsteps are playing faster without any weapons (As Gordon)

### ISSUE THREAD

**Issue by GrubPL** [SonicEraZoR/Portal-Base#25]
so ive decided to make an portal 1 mod usin this base, tho i noticed an issue, so footsteps just behave weird, they are just playing faster, seems to be tied to the faster walkin animation without weapons, it inst a major bug, tho a bug non the less, asspesially something thats an base for portal mods, issue only applies on the gordon freeman playermodel, my guess its that the sounds are in the male animations file.

**Comment by GuestSneezeOSDev** [comment]
This project is no longer active and I recommend checking out this alternative instead [Source SDK Base Ultimate](https://github.com/AlmightyToastz/Source-SDK-Base-Ultimate)

**Comment by SonicEraZoR** [comment]
Yeah, those faster footsteps sounds are tied to the model animation, so I'm pretty sure it's not fixable through code and the model itself needs to be edited. I might edit the model sometime later.
